### PR TITLE
 Add instructions for a non-cncf project/org.

### PR DESCRIPTION
**Pull request by isamrish**

### CODE REVIEW

File: SYNC.md [hunk 1]
```diff
@@ -26,3 +26,59 @@
 3. Add "name,company,login" CSV header.
 4. Example [file](https://docs.google.com/spreadsheets/d/1QShmHcStYh5BjTjdOAASFK9V4WaYwJSFu1maBdcV5YA/edit#gid=0)
 4. Run `./maintainers.sh` script. Follow its instructions.
+
```
Comment (lukaszgryglicki) [inline SYNC.md:29]: I would love to add this description but please create a separate md file and put a link to this file in SYNC.md or in README.md or in both.
I just want to have more smaller md files.
Thanks.

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]

File: SYNC.md (hunk 1 above)
Comment (isamrish) [reply SYNC.md:29]: sure, Thanks

**Comment by isamrish** [comment]
@lukaszgryglicki  I have seperated instructions in ADD_PROJECT.md file. Also, I have added links to both README.md and SYNC.md file.

**Comment by lukaszgryglicki** [comment]
Merged, thanks.

**Pull request merged**